### PR TITLE
konflux(prod): add toolchain host and member OLM configuration

### PR DESCRIPTION
**Pull request by mfrancisc**
Move the OLM configuration for host and member operator one level up , so that it get's deployed to both Stage and Prod via ArgoCD.

Jira: https://issues.redhat.com/browse/ASC-496

### CODE REVIEW

File: components/sandbox/toolchain-host-operator/base/olm/toolchain-host-operator.yaml
Comment (hugares) [inline components/sandbox/toolchain-host-operator/base/olm/toolchain-host-operator.yaml:1]: You need to remove `- toolchain-host-operator.yaml` from `components/sandbox/toolchain-host-operator/staging/stone-stage-p01/kustomization.yaml`

File: components/sandbox/toolchain-host-operator/staging/stone-stg-host/toolchain-host-operator.yaml
Comment (hugares) [inline components/sandbox/toolchain-host-operator/staging/stone-stg-host/toolchain-host-operator.yaml:1]: You need to remove `- toolchain-host-operator.yaml` from `components/sandbox/toolchain-host-operator/staging/stone-stg-host/kustomization.yaml`

**Review by hugares — CHANGES_REQUESTED** [review]

File: components/sandbox/toolchain-host-operator/base/olm/toolchain-host-operator.yaml
Comment (mfrancisc) [reply components/sandbox/toolchain-host-operator/base/olm/toolchain-host-operator.yaml:1]: done https://github.com/redhat-appstudio/infra-deployments/pull/3342/commits/ef72be689623960190f68bcccc016da3bc7bd57b thanks!

**Review by hugares — APPROVED** [review]
/lgtm

**Comment by mfrancisc** [comment]
Not sure I understand what exactly failed in the e2e tests: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/redhat-appstudio_infra-deployments/3342/pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests/1764412885297008640/build-log.txt

Seems like registration service deployment get's deleted? 😕

**Comment by mfrancisc** [comment]
@hugares @dperaza4dustbit I had to rebase , could you PTAL and /lgtm if looks ok ?

**Review by dperaza4dustbit — APPROVED** [review]
/lgtm

**Pull request merged**